frontend/registration.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+
+    <!-- SEO meta tags -->
+    <meta
+      name="description"
+      content="This is a registration page created by Team PASS: an undergraduate Gemstone research team from UMD studying cybersecurity."
+    />
+    <meta name="author" content="Team Pass" />
+
+    <!-- CSS -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="./assets/styles/main.css" type="text/css" />
+
+    <title>Team PASS | Registration</title>
+    <link rel="icon" type="image/png" href="favicon.png" />
+  </head>
+  <body>
+    <main class="container">
+      <img
+        src="./assets/images/team-logo.svg"
+        alt="Team PASS Logo"
+        class="team-logo"
+      />
+      <section class="row justify-content-center">
+        <div class="col-6 card">
+          <div class="card-body">
+            <h3 class="card-title">Registration</h3>
+
+            <!--Login Form-->
+            <form>
+              <div class="form-group">
+                <label for="email">Email address</label>
+                <input
+                  type="email"
+                  class="form-control"
+                  id="email"
+                  aria-describedby="emailHelp"
+                />
+                <small id="emailHelp" class="form-text text-muted"
+                  >We'll never share your email with anyone else.</small
+                >
```

Comment:
We should delete this line (it's not really needed, and I already removed it in my PR).
```suggestion
                />
```